### ISSUE TITLE
Hi sir but i was using your extension but their icons were vanished the screenshot might explain for itself

### ISSUE THREAD

**Issue by RudraAsthana** [GorvGoyl/Shortcut-Menu-Bar-VSCode-Extension#55]
![image](https://user-images.githubusercontent.com/79319354/117656912-8e083980-b1b6-11eb-870a-a347aad2fd9f.png)

**Comment by GorvGoyl** [comment]
yes, it's an issue with VSCode latest release. closing as duplicate of https://github.com/GorvGoyl/Shortcut-Menu-Bar-VSCode-Extension/issues/54
